RecoLocalCalo/EcalRecProducers/python/ecalRecHit_cfi.py

```diff
@@ -93,4 +93,17 @@
                  recoverEBFE = False,
                  recoverEEFE = False,
                  recoverEBIsolatedChannels = False
-                  )
+                )
+
+# Phase 2 modifications
+from Configuration.Eras.Modifier_phase2_ecal_devel_cff import phase2_ecal_devel
+phase2_ecal_devel.toModify(ecalRecHit,
+    EBuncalibRecHitCollection = cms.InputTag("ecalUncalibRecHitPhase2", "EcalUncalibRecHitsEB"),
+    EEuncalibRecHitCollection = cms.InputTag(""),  # No EE input since there is no ECAL endcap in Phase 2
+    killDeadChannels = cms.bool(False),
+    recoverEBFE = cms.bool(False),
+    recoverEEFE = cms.bool(False),
+    recoverEBIsolatedChannels = cms.bool(False),
+    recoverEEIsolatedChannels = cms.bool(False)
```

Comment:
```suggestion
    EBuncalibRecHitCollection = "ecalUncalibRecHitPhase2:EcalUncalibRecHitsEB",
    EEuncalibRecHitCollection = "",  # No EE input since there is no ECAL endcap in Phase 2
    killDeadChannels = False,
    recoverEBFE = False,
    recoverEEFE = False,
    recoverEBIsolatedChannels = False,
    recoverEEIsolatedChannels = False
```

Reply:
What is the reason behind this suggestion?

Reply:
avoid duplicating the types when they are already specified elsewhere.

https://cms-sw.github.io/cms_coding_rules.html 6.17
"When one customizes an existing parameter in clone(), Modifier.toModify(), or in assignment, explicit types on the right hand side should be avoided."

Reply:
Just to add the benefit of avoiding the types, then the framework checks that those parameters already exist, and e.g. in case of typos the you get an error (instead of unexpected results).

Reply:
Thanks. Will change this.